types/node/test/http2.ts

```diff
@@ -321,14 +321,14 @@ import { URL } from 'node:url';
         response.write(Buffer.from([]), (err: Error) => {});
         response.write(Buffer.from([]), 'utf8');
         response.write(Buffer.from([]), 'utf8', (err: Error) => {});
-        response.end();
-        response.end(() => {});
-        response.end('');
-        response.end('', () => {});
-        response.end('', 'utf8');
-        response.end('', 'utf8', () => {});
-        response.end(Buffer.from([]));
-        response.end(Buffer.from([]), () => {});
+        response.end().end();
```

Comment:
Wouldn't it be a cleaner test if all of these lines were chained instead of just appending a `.end()` call?